templates/deploy-to-existing-k8s.yml

```diff
@@ -12,19 +12,12 @@ variables:
 
   # Container registry service connection established during pipeline creation - Docker Hub/ACR/...
   dockerRegistryServiceConnection: '{{ containerRegistryConnection.Id }}'
-  imageRepository: '$(build.repository.name)'
-  containerRegistryName: '{{ containerRegistryName }}'
-
-  dockerfilePath: '**/Dockerfile'
-
-  # Kubernetes Environment
-  environmentName: '{{ environment.Environment.Name }}'
-  k8sResourceName: "{{ environment.Resources.kubernetesConnection.Name }}"
-  k8sResourceNamespace: "{{ environment.Resources.kubernetesConnection.Namespace }}"
-
-    # Docker Image Name
   imageName: '{{ imageName }}'
+  containerRegistryName: '{{ containerRegistryName }}.azurecr.io'

```

Comment:
Why this change? This way we'll have to change the template again when we add support in UX for choosing container registry other than ACR